public/index.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>2048 Clone</title>
+  <link rel="icon" type="image/x-icon" href="public/icons/grid-filled-32.png">
+  <style>
+    .grid-3x3 {
+      display: grid;
+      grid-template-columns: 1fr 5fr 1fr;
+      grid-template-rows: 1fr 5fr 1fr;
+    }
+
+    header,
+    footer {
+      text-align: center;
+    }
+  </style>
+</head>
+<body>
+  <article>
+    <header>
+      <h1>
+        This is a clone 2048 game
+      </h1>
+    </header>
+    <section id="main">
+      <game-app class="grid-3x3" size="4"></game-app>
+    </section>
+    <footer>
+      <!-- reset button -->
+      <button type="button" id="reset" class="btn reset js-reset-btn">Reset</button>
+    </footer>
+  </article>
+  
+  <!-- <script src="src/components/Board.js" type="module"></script>
+  <script src="src/components/Cell.js" type="module"></script>
+  <script src="src/components/CustomModal.js" type="module"></script> -->
+  <script src="public/web.js" type="module"></script>
+  <script type="text/javascript">
+    // document.addEventListener('DOMContentLoaded', () => {
+    //   var testing = 'bloop'
```

Comment:
haha. did your testing work?

Reply:
it showed me some interesting stuff! but didn't give me the answers I was looking for